lib/http-bridge/test/integration/Test/Integration/Scenario/CLISpec.hs

```diff
@@ -0,0 +1,118 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Integration.Scenario.CLISpec
+    ( specNoCluster
+    , specWithCluster
+    ) where
+
+import Prelude
+
+import Cardano.Wallet.Api.Types
+    ( ApiWallet )
+import Control.Monad
+    ( forM_ )
+import Data.List
+    ( length )
+import Data.List.Split
+    ( splitOn )
+import Data.Text
+    ( Text )
+import Test.Hspec.Expectations.Lifted
+    ( shouldBe
+    , shouldContain
+    , shouldNotContain
+    )
+import System.Command
+    ( Stdout (..)
+    , command
+    )
+import Test.Hspec
+    ( SpecWith
+    , describe
+    , it
+    )
+import Test.Integration.Framework.DSL
+    ( Context (..)
+    , Headers (..)
+    , Payload (..)
+    , expectResponseCode
+    , getFromResponse
+    , json
+    , request
+    , walletId
+    )
+
+import qualified Network.HTTP.Types.Status as HTTP
+import qualified Data.Text as T
+
+specNoCluster :: SpecWith ()
+specNoCluster = do
+
+    it "CLI - Shows version" $  do
+        Stdout out <- command [] "cardano-wallet" ["--version"]
+        cabal <- readFile "../../cardano-wallet.cabal"
+        cabal `shouldContain` out
+
+    it "CLI - Can generate mnemonics with default size" $  do
+        Stdout out <- command [] "cardano-wallet" ["mnemonic", "generate"]
+        length (splitOn " " out) `shouldBe` 15
```

Comment:
You're probably just looking for: https://hackage.haskell.org/package/base-4.11.1.0/docs/Data-List.html#v:words from the `base` package. No need for the extra `split` package here :pray: